demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -746,6 +748,17 @@ export class DemoMeetingApp
       );
     });
 
+    if(!this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback())  {
```

Comment:
Do we need to check for unified plan here like in other places?

Reply:
I added a reply to this question up above as well:

"I do not have to check for unified plan as far as I understand. The code snippet shared is the one used for checking for Segmentation processor. In my code, I am just checking whether the browser supports canvas capture playback. This check is done because iOS browsers will not be able to support video filters as they lack canvas capture stream playback."

Reply:
We check for enableUnifiedPlanForChromiumBasedBrowsers in other filter population. 
```
    if (
      this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback() &&
      this.enableUnifiedPlanForChromiumBasedBrowsers
    ) {
```

Reply:
I have included the check for unified plan.